website/blog/2025-06-04-jest-30.md

```diff
@@ -0,0 +1,181 @@
tonyd33](https://github.com/tonyd33), [@madcapnmckay](https://github.com/madcapnmckay), [@dongwa](https://github.com/dongwa), [@gagan-bhullar-tech](https://github.com/gagan-bhullar-tech), [@ikonst](https://github.com/ikonst), [@ZuBB](https://github.com/ZuBB), [@jzaefferer](https://github.com/jzaefferer), [@brandonnorsworthy](https://github.com/brandonnorsworthy), [@henny1105](https://github.com/henny1105), [@DmitryMakhnev](https://github.com/DmitryMakhnev), [@askoufis](https://github.com/askoufis), [@RahulARanger](https://github.com/RahulARanger), [@Jon-Biz](https://github.com/Jon-Biz), [@fynsta](https://github.com/fynsta), [@KonnorRogers](https://github.com/KonnorRogers), [@BondarenkoAlex](https://github.com/BondarenkoAlex), [@mouadhbb](https://github.com/mouadhbb), [@kemuridama](https://github.com/kemuridama), [@Avi-E-Koenig](https://github.com/Avi-E-Koenig), [@davidroeca](https://github.com/davidroeca), [@akwodkiewicz](https://github.com/akwodkiewicz), [@mukul-turing](https://github.com/mukul-turing), [@dnicolson](https://github.com/dnicolson), [@colinacassidy](https://github.com/colinacassidy), [@ofekm97](https://github.com/ofekm97), [@haze](https://github.com/haze), [@Vadimchesh](https://github.com/Vadimchesh), [@peterdenham](https://github.com/peterdenham), [@ShuZhong](https://github.com/ShuZhong), [@manoraj](https://github.com/manoraj), [@nicolo-ribaudo](https://github.com/nicolo-ribaudo), [@georgekaran](https://github.com/georgekaran), [@MathieuFedrigo](https://github.com/MathieuFedrigo), [@hkdobrev](https://github.com/hkdobrev), [@Germandrummer92](https://github.com/Germandrummer92), [@CheadleCheadle](https://github.com/CheadleCheadle), [@notaphplover](https://github.com/notaphplover), [@danbeam](https://github.com/danbeam), [@arescrimson](https://github.com/arescrimson), [@yepitschunked](https://github.com/yepitschunked), [@JimminiKin](https://github.com/JimminiKin), [@DerTimonius](https://github.com/DerTimonius), [@vkml](https://github.com/vkml), [@ginabethrussell](https://github.com/ginabethrussell), [@jeremiah-snee-openx](https://github.com/jeremiah-snee-openx), [@WillianAgostini](https://github.com/WillianAgostini), [@casey-lentz](https://github.com/casey-lentz), [@faizanu94](https://github.com/faizanu94), [@someone635](https://github.com/someone635), [@rafaelrabelos](https://github.com/rafaelrabelos), [@RayBrokeSomething](https://github.com/RayBrokeSomething), [@DaniAcu](https://github.com/DaniAcu), [@mattkubej](https://github.com/mattkubej), [@tr1ckydev](https://github.com/tr1ckydev), [@shresthasurav](https://github.com/shresthasurav), [@the-ress](https://github.com/the-ress), [@Mutesa-Cedric](https://github.com/Mutesa-Cedric), [@nolddor](https://github.com/nolddor), [@alexreardon](https://github.com/alexreardon), [@Peeja](https://github.com/Peeja), [@verycosy](https://github.com/verycosy), [@mknight-atl](https://github.com/mknight-atl), [@maro1993](https://github.com/maro1993), [@Eric-Tyrrell22](https://github.com/Eric-Tyrrell22)
```

Comment:
I'm not sure how the list is generated here, but my first contributing PR is #15618, 2 weeks ago. 😜

Maybe that's because I'm already been mentioned at `Performance & Memory Improvements` section? 😁

Reply:
Yes you are mentioned above already.